coder_sniffer/Drupal/Sniffs/Semantics/FunctionTSniff.php

```diff
@@ -75,7 +75,10 @@ public function processFunctionCall(
             return;
         }
 
-        if ($tokens[$argument['start']]['code'] !== T_CONSTANT_ENCAPSED_STRING) {
+        if ($tokens[$argument['start']]['code'] !== T_CONSTANT_ENCAPSED_STRING
+            && $tokens[$argument['start']]['code'] !== T_STRING
+            && $tokens[$argument['start']]['code'] !== 'PHPCS_T_SELF'
```

Comment:
why are your referring to the string here? This can be T_SELF, which is defined in PHPCS, right?

Reply:
Right, fixed.